src/content/lib/ua_overrider.jsm

```diff
@@ -59,34 +58,7 @@ class UAOverrider {
 
   getUAForURI(uri) {
     let bareUri = uri.specIgnoringRef;
```

Comment:
bareUri is not used, please remove unused variable.
As bareUri is removed, I think we don't need getUAForURI just for calling lookupUAOverride.